oraichain/assetlist.json

```diff
@@ -301,6 +317,10 @@
       "symbol": "USDC",
       "images": [
         {
+          "image_sync": {
+            "chain_name": "ethereum",
+            "base_denom": "uusdc"
```

Comment:
not the correct base. The base should be right above, in the traces. It's an address of some kind.
Same goes for USDT